legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/tests/testRelationalExtension.pure

```diff
@@ -176,12 +176,30 @@ Class meta::relational::tests::runtime::extractDBs::SubstitutionTestClass1
    prop1: Integer[1];
 }
 
+Class meta::relational::tests::runtime::extractDBs::TestClassWithBoolProperty
+{
+   prop1: Boolean[1];
+}
+
+Class meta::relational::tests::runtime::extractDBs::TestClassWithDateProperty
+{
+   prop1: Date[1];
+}
+
 ###Relational
 Database meta::relational::tests::runtime::extractDBs::SubstitutionTestDB1_Inc
 (
    Table testTable1(prop1 INT)
 )
 
+###Relational
+Database meta::relational::tests::runtime::extractDBs::TestDifferentTypesDB
+(
+   Table testTableFloat(prop1 FLOAT)
+   Table testTableTimestamp(prop1 TIMESTAMP)
+   Table testTableBit(prop1 BIT)
+)
+
 ###Relational
```

Comment:
can you write a test involving enum and enumeration mapping too

Reply:
we may also have to consider joins, and see what happens with them, and accordingly update our logic

Reply:
Other than these 2 things, code looks good

Reply:
Code is for RelationalPropertyMapping only, so it won't touch enumerationMappings.
I have written one test for join, it's working fine.